internal/service/batch/job_queue_schema.go

```diff
@@ -29,7 +29,7 @@ func jobQueueSchema0(ctx context.Context) schema.Schema {
 			"arn": framework.ARNAttributeComputedOnly(),
 			"compute_environments": schema.ListAttribute{
 				ElementType: types.StringType,
-				Required:    true,
+				// Required:    true,
```

Comment:
Can we remove this instead of commenting?
```suggestion
```

Reply:
good catch. this should still be included - adding back now!